### DISCUSSION TITLE
Warning: input is not rfc822 compliant: not in < bracket

### DISCUSSION THREAD

**Discussion by c1am** [php-mime-mail-parser/php-mime-mail-parser#394] [Q&A]
I'm getting this error when trying to parse an email. Different parts will give different errors such as:
Warning: input is not rfc822 compliant: not in < bracket
Warning: input is not rfc822 compliant: already in < bracket

I'm wondering if there is a fix for this?

**Comment by eXorus** [comment]
If your email is not compliant with RFC822 I can't really do anything.
Do you have the email raw to investigate?

**Reply by c1am** [reply]
I can't give you the email, but is there a way for me to check myself what is breaking it?
Additionally, do you know of a workaround like a check or fixer for RFC822 compliancy before being parsed?

**Reply by eXorus** [reply]
You can give the email and remove sensitive information and keep the structure.

The error is pretty clear you have a header that uses `<<something>>` instead of `<something>`

You can use the middleware to fix this kind of issue.
